### ISSUE TITLE
License information. 

### ISSUE THREAD

**Issue by Mosaddeq107** [ubsuny/23-Homework4G5#48]
Guys, do you have any license? We want to copy some codes from your repository. Is it permissible according to your license?

**Comment by tirthbha** [comment]
@Mosaddeq107 Yes, but we haven't check the compatibility of our license for this task yet. BTW, which code you want to copy from our repository?
